src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements Drawable, AreaCalculable {
+    private double radius;
+
+    public Circle(double radius, String color) {
+        super.setColor(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double area() {
+        return Math.PI * Math.pow(radius,2);
```

Comment:
All magic numbers in your code should be constants. Consider defining 'Math.PI' as a constant field in your class, with a clear name indicating its purpose.